integration-tests/src/test/resources/ssl/SslTestClient.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+import java.util.Hashtable;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import javax.jms.ConnectionFactory;
+
+public class SslTestClient {
+  private String    url;
+
+  public SslTestClient(String[] args)

```

Comment:
Why do we call it ssl client? there the ssl invoke happen?

Reply:
It takes t3s URL to get initial context.